### DISCUSSION TITLE
[Implemented] Specification for `plotResidualsVsTime()` and `plotResidualsVsSimulated()`

### DISCUSSION THREAD

**Discussion by IndrajeetPatil** [Open-Systems-Pharmacology/OSPSuite-R#811] [Ideas]
Related to #674 

# Signature

```r
# `...` will be parameters to control annotations and aesthetics of the plot
plotResidualsVsTime(dataCombined, ...)
```

# Implementation plan

```r
data <- .calculateResiduals()

tlf::plotResVsTime(data, ...) 
tlf::plotResVsPred(data, ...) 
```

# Rough expected output:

![image](https://user-images.githubusercontent.com/11330453/157641883-753abd99-8285-4440-ba8b-c1b7ee1f4d1f.png)

# Decision points:

N.B. The option already rejected by the team is scratched out.

- [x] Nature of the needed function to extract needed dataframe: Member method versus ~~function~~
- [x] Data format returned by respective function: ~~wide~~ versus tidy

# Tasks

Each task will get its own PR.

- Implement `$calculateResiduals()` function (#803)
- Implement `plotResidualsVsTime()`

# Ditto for `plotResidualsVsSimulated()`

**Comment by IndrajeetPatil** [comment]
@PavelBal Do the same concerns mentioned for `plotObservedVsSimulated()` also apply to this function? 
I am guessing yes, but just wanna double-check.

**Reply by IndrajeetPatil** [reply]
@PavelBal Referring to [this comment](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/discussions/809#discussioncomment-2713748).

**Comment by PavelBal** [comment]
Yes

**Comment by IndrajeetPatil** [comment]
@PavelBal The default scale for this plot should be linear, and not logarithmic, correct?

**Reply by IndrajeetPatil** [reply]
Never mind. Already addressed.
